challenge.md

```diff
@@ -31,21 +31,15 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
- 
- <!-- Answer Ends Here -->
+absolute. because it can be access from anywhere
```

Comment:
Right answer, but your reasoning is off. It's an absolute path because it defines the location of `Desktop` with respect to the root (/), a fixed location in the computer. Consequently, no matter where we are in our filesystem, this path will always be accurate.